src/main/java/frc/robot/RobotConstants.java

```diff
@@ -18,6 +18,17 @@
         public final static Double LIFT_TUBE_CIRCUMFERENCE = LIFT_TUBE_DIAMETER * Math.PI;
         /** */
         public final static Double DISTANCE_LIFT_MECHANISM_FROM_FLOOR = 32.24;
+        /**
+         * the angles which the oneEighty subsystem should do a PID on.
+         */
+        public static enum Angle{
+            //TODO: Set real angles.
+            kHatch(0), kCargo(180), kFeeder(270);
+            public double key;
```

Comment:
Maybe some documentation explaining what each item is used for.

Reply:
what about the following cases:
what angle for collecting the cargo?
when we have to "shoot" the ball into the highest target on the rocket?